### PR TITLE
 Implemented opt-out feature for postgres json serialization

### PR DESCRIPTION
**Pull request by bemueller**
No need to define dumb coder in each project that is using activerecord-typedstore .

### CODE REVIEW

File: lib/active_record/typed_store/extension.rb [hunk 1]
```diff
@@ -21,7 +21,11 @@ module ClassMethods
       def typed_store(store_attribute, options={}, &block)
         dsl = DSL.new(options.fetch(:accessors, true), &block)
 
-        serialize store_attribute, create_coder(store_attribute, dsl.columns).new(options[:coder])
+        unless options[:no_coder]
```
Comment (byroot) [inline lib/active_record/typed_store/extension.rb]: I think `coder: false` would be better.

**Comment by byroot** [comment]
Also could you update the tests please?

**Comment by bemueller** [comment]
I couldn't get the tests running yesterday (sqlite3 gem is not working correctly). So I made a _good guess_...

**Comment by byroot** [comment]
Master build was broken indeed :/ Sorry for that.

I rebased you branch on top of a fixed master: https://travis-ci.org/byroot/activerecord-typedstore/builds/108555062

Let's see.

**Comment by byroot** [comment]
Ok, build is green, merged as ee3065f50c6220324e087ec1c01f40f9d5dc2cf2

Thanks!

**Pull request closed without merging**